package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "movie-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --mode development --open --hot",
```

Comment:
As far as I know, the newest webpack's official way to start a dev-server is just `webpack serve`. So your script would look something like `webpack serve --mode development --open --hot`

Also, the majority of configuration options should be set in config file, not through cli arguments. So, please move them to config file, and leave here only the necessary ones. Probably, the only necessary thing would be an env variable.

Reply:
Seems like you completely removed `open` and `hot` configurations. However, what I meant was to keep them in the webpack's config instead. Please, include them in webpack's config for development mode.

Reply:
fixed